src/parser/deathknight/blood/modules/features/BoneShield.js

```diff
@@ -43,7 +45,7 @@ class BoneShield extends Analyzer {
     when(this.uptimeSuggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => suggest('Your Bone Shield uptime can be improved. Try to keep it up at all times.')
```

Comment:
These are also important to internationalize

Reply:
Do these just get wrapped in Trans tags?

Reply:
It doesn't matter for strings, though using Trans everywhere might be easier.

If you can fix the conflicts for this PR I'd like to merge this before that though.

Reply:
Done!